boltdesignsystem/bolt
# For the love of humanity, make Pattern Lab docs at least legible.

## Jira

N/A

## Summary

Restructure component docs to have better information architecture.

## How to test

Run the branch locally and go through each page in Pattern Lab. Make sure there are no broken layouts.
